frontend/CLAUDE.md

```
@@ -504,6 +504,11 @@ const routes = {
 
 ### Components
 
+#### Steward Review Components (`src/components/`)
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)